src/osp/Active/SysWire.cpp

```diff
@@ -51,21 +48,6 @@ void WireInput::doErase()
      list()->cut(this);
 };
 
-SysWire::SysWire(ActiveScene &scene)
- : m_updateWire(scene.get_update_order(), "wire", "", "",
-                [this] (ActiveScene& rScene) { this->update_propagate(rScene); })
-{
-
-}
-
-
-void SysWire::update_propagate(ActiveScene& rScene)
-{
-    for (DependentOutput &output : m_dependentOutputs)
-    {
-        output.m_element->propagate_output(output.m_output);
-    }
-}
```

Comment:
This doesn't actually do anything. All of the propagate_output functions are blank or just prints. Wiring system needs a ton of redesigning too.